prow/plugins/plugins.go

```diff
@@ -371,11 +376,11 @@ func (c *Configuration) TriggerFor(org, repo string) *Trigger {
 }
 
 func (c *Configuration) setDefaults() {
-	if c.ConfigUpdater.ConfigFile == "" {
-		c.ConfigUpdater.ConfigFile = "prow/config.yaml"
-	}
-	if c.ConfigUpdater.PluginFile == "" {
-		c.ConfigUpdater.PluginFile = "prow/plugins.yaml"
+	if len(c.ConfigUpdater.Maps) == 0 {
+		c.ConfigUpdater.Maps = map[string]string{
+			"prow/config.yaml":  "config",
+			"prow/plugins.yaml": "plugins",
+		}
```

Comment:
don't need this block anymore?

Reply:
Keeping it for backwards compat?

Reply:
there are checks for `ConfigFile` and `PluginFile` explicitly though

Reply:
Done, moved all the ConfigFile PluginFile backwards compat in here